language/404.md

```diff
@@ -0,0 +1,24 @@
+---
+type: 
+title: "404 - We're sorry. Looks like that page doesn't exist."
+---
+
+Try one of these instead:
+
+### Principles
+
+Watson design principles provide the foundation for designing excellent cognitive experiences for users.
+
+[Principles <span class="icon-next"></span>](/principles)
+
+### Guidelines
+
+These style, interaction, and branding guides offer detailed recommendations for elements like color, typography, and motion.
```

Comment:
Can we change the examples to line up with the guides above? Recommending changing this to: "elements like color, motion, and wordmark."